src/panels/RetinaCapturePanel.ts

```diff
@@ -0,0 +1,169 @@
+import { Uri, window, workspace } from "vscode";
+import { BasePanel, PanelDataProvider } from "./BasePanel";
+import { InitialState, ToVsCodeMsgDef } from "../webview-contract/webviewDefinitions/retinaCapture";
+import { MessageHandler } from "../webview-contract/messaging";
+import { TelemetryDefinition } from "../webview-contract/webviewTypes";
+import { withOptionalTempFile } from "../commands/utils/tempfile";
+import { KubectlVersion, invokeKubectlCommand } from "../commands/utils/kubectl";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { failed } from "../commands/utils/errorable";
+import * as vscode from "vscode";
+import { longRunning } from "../commands/utils/host";
+import { relative } from "path";
+import { platform } from "os";
+import open from 'open';
```

Comment:
Do we need to add this as a dependency in `package.json`?

Reply:
Thanks for this question. This comes bundles within one of the packages, so never required to require this.